toolchain/check/context.cpp

```diff
@@ -453,7 +453,7 @@ struct ProhibitedAccessInfo {
 };
 
 auto Context::AppendLookupScopesForConstant(
-    SemIRLoc loc, SemIR::ConstantId base_const_id,
+    SemIR::LocId loc, SemIR::ConstantId base_const_id,
```

Comment:
```suggestion
    SemIR::LocId loc_id, SemIR::ConstantId base_const_id,
```

Is it worth updating the name to reflect the type change?